docs/app/Examples/collections/Menu/Types/Attached.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Dropdown, Icon, Menu, Segment } from 'stardust'
+
+// TODO: Fix <Dropdown> usage
+// TODO: Update <Search> usage after its will be implemented
+
+const Attached = () => {
+  return (
+    <div>
+      <Menu attached='top'>
+        <Menu.Item as={Dropdown} icon='wrench'>
+          <Dropdown.Menu>
+            <Dropdown.Item>
+              <Icon name='dropdown icon' />
+              <span className='text'>New</span>
+
+              <Dropdown.Menu>
```

Comment:
The Dropdown has very limited support for custom markup.  See #416 for more.  It is best to use a Dropdown with an `options` prop instead of sub components for now.

We can leave this PR as is since it is since you've already put so much into it.